myapp/spec/system/admin/tasks_spec.rb

```diff
@@ -66,14 +66,14 @@ def log_in(user)
       expect(page).to have_content(I18n.t 'msg_update_success')
       expect(page).to have_content("Edited Task")
     end
-    it "allows admin to reassign task" do
-      visit edit_task_path(locale: I18n.locale, id: task.id)
+    # it "allows admin to reassign task" do
```

Comment:
[Okay starting next time] 
For permanent deletions, please remove the code instead of commenting it out.
If you want to temporarily skip case, please use `skip` with a reason like below.

```ruby
it 'allows admin to reassign task', skip: 'Temporarily skipped due to xxx' do
```